examples/archive.html

```diff
@@ -73,6 +73,16 @@ <h3 id="offcanvasRightLabel">Images</h3>
         <p id="response" class="mb-5">No link = No images...</p>
         <div id="imgContainer"></div>
       </div>
+        <hr>
+      <div class="offcanvas-footer d-flex justify-content-evenly align-items-center">
```

Comment:
Hi, just wanted to know - how does this look if there is only one page? Thank you!

Reply:
Hello @jywarren 
If you check out the second GIF, that is a one-paged pagination
Do you have any suggestions as to a different look?